p2p/gossip.go

```diff
@@ -22,6 +22,8 @@ type GossipMessage struct {
 // GossiperOption sets optional parameters of Gossiper.
 type GossiperOption func(*Gossiper) error
 
+type NewMessage func(msg *GossipMessage)
```

Comment:
can we rename this `GossipMessageHandler`

Reply:
done